update nodejs to v10 (9) and gradle to v4.10.3 (4.1.3)

Phoscur opened the pull request:
got some more updates :D

marcomaisel commented:
Did you test these changes? Your last merge request causes the build process on dockerhub to fail. I don't use this image at the moment so I don't have the time to debug it right now, sorry

Phoscur commented:
In a way yes, I added the sdkmanager line with "echo y | " in my gitlab-ci build and that made the build go through (see #3) this is due to the new default version 26 and its new license from 19.1.2019).
But the image itself? No I was about to start getting my docker env to work so I could test and publish images myself when you merged my request (in which an upgrade of the android-tools might still be missing).

For this one I did not test the image either no. But I checked the versions and commands I updated in this pull request.
